README.md

```diff
@@ -560,6 +560,12 @@ There is a PrimeNumberResource that checks whether an integer is prime or not. T
 
 Where `{uniqueId}` is an unique identifier that is calculated at startup time to uniquely identify the metrics of the application.
 
+### `micrometer/prometheus-kafka`
+
+Verifies that the kafka metrics are exposed in the embedded Prometheus instance provided by OpenShift.
```

Comment:
Do you need additional user privileges to get data from Prometheus instance?
I remember some discussion about that on our chat.

Reply:
Yes, I had added this information on the prerequisites section, but I've also added a link about this in these modules.